artifactory/manager.go

```diff
@@ -160,6 +160,11 @@ func (sm *ArtifactoryServicesManagerImp) GetPermissionTarget(permissionTargetNam
 }
 
 func (sm *ArtifactoryServicesManagerImp) PublishBuildInfo(build *buildinfo.BuildInfo, projectKey string) error {
+	_, err := sm.PublishBuildInfoWithSummary(build, projectKey)
+	return err
+}
+
+func (sm *ArtifactoryServicesManagerImp) PublishBuildInfoWithSummary(build *buildinfo.BuildInfo, projectKey string) (*services.BuildPublishSummary, error) {
```

Comment:
PublishBuildInfoWithSummary Is not needed - we'd prefer that PublishBuildInfo will return the summary.
```suggestion
func (sm *ArtifactoryServicesManagerImp) PublishBuildInfo(build *buildinfo.BuildInfo, projectKey string) (*services.BuildPublishSummary, error) {
```